src/Lecture2.hs

```diff
@@ -97,7 +115,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: String -> String
+dropSpaces string = takeWhile (not . Data.Char.isSpace) (dropWhile Data.Char.isSpace string)
```

Comment:
You don't need to use a fully-qualified function name because your import is implicit. Also, you can eta-reduce the function after you use function composition here:

```suggestion
dropSpaces = takeWhile (not . isSpace) . dropWhile isSpace
```